numba/cuda/tests/cudapy/test_powi.py

```diff
@@ -24,6 +25,31 @@ def cu_mat_power_binop(A, power, power_A):
     power_A[y, x] = A[y, x] ** power
 
 
+def vec_pow(r, x, y):
+    i = cuda.grid(1)
+
+    if i < len(r):
+        r[i] = pow(x[i], y[i])
+
+
+def vec_pow_binop(r, x, y):
+    i = cuda.grid(1)
+
+    if i < len(r):
+        r[i] = x[i] ** y[i]
+
+
+def vec_pow_inplace_binop(r, x):
+    i = cuda.grid(1)
+
+    if i < len(r):
+        r[i] **= x[i]
+
+
+def random_complex(N):
+    return (np.random.random(1) + np.random.random(1) * 1j)
```

Comment:
For reproducibility, probably a good idea to set the RNG seed?

Reply:
Done!